android/src/main/java/com/brentvatne/receiver/AudioBecomingNoisyReceiver.kt

```diff
@@ -0,0 +1,38 @@
+package com.brentvatne.receiver
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.media.AudioManager
+import androidx.core.content.ContextCompat
+
+class AudioBecomingNoisyReceiver(private val context: Context) : BroadcastReceiver() {
+    private var listener: BecomingNoisyListener = BecomingNoisyListener.NO_OP
+
+    init {
+        context.applicationContext
+    }
```

Comment:
I think we want to do it that way (but please double check it)
```suggestion
class AudioBecomingNoisyReceiver(private val context: Context) : BroadcastReceiver() {
    private var listener: BecomingNoisyListener = BecomingNoisyListener.NO_OP
    private var context = context.applicationContext
```

Reply:
It was a good point.